package-lock.json

```diff
@@ -13465,9 +13461,15 @@
       }
     },
     "icu4c-data": {
+<<<<<<< HEAD
```

Comment:
It seems like your committed some unrelated changes to this file

Reply:
Yes. can you tell me how to remove the package.json file from this PR.

Reply:
Never mind I have removed the file. Please check now. @hanzei